frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java

```diff
@@ -30,18 +32,35 @@ public FetchBackgroundable(Project project,
       String refspec,
       String taskTitle,
       String failureNotificationText,
-      String successNotificationText) {
+      String successNotificationText,
+      @Nullable String taskSubtitle) {
```

Comment:
Hmm this is `null` only for the Pull task currently? or anything else?

Reply:
If only for pull, then maybe you could give `Pulling...` there instead of the default, and the other c'tor isn't even needed

Reply:
No, this is also for Slide-in (in case of remote branch)

Reply:
Hmm ok so in case of Slide-in it's still `Fetching...` that shows up right? :thinking:

Reply:
Yes, it's true. IMHO its appropriate message in that case - fetching is what actually this action do.